tests/3-unit/GlpiPlugin/Formcreator/Field/TextareaField.php

```diff
@@ -168,7 +168,7 @@ public function testDeserializeValue($question, $input, $expected) {
 
       $instance->parseAnswerValues($input);
       $instance->deserializeValue($input[$key]);
-      $output = $instance->getValueForTargetText('', false);
+      $output = $instance->getValueForTargetText('', true);
```

Comment:
the change in the expected value reveals that this bool was wrongly set to false.